EventFilter/SiPixelRawToDigi/python/SiPixelRawToDigi_cfi.py

```diff
@@ -14,6 +14,17 @@
     UsePhase1 = True
 )
 
+from Configuration.ProcessModifiers.siPixelQualityRawToDigi_cff import siPixelQualityRawToDigi
+siPixelQualityRawToDigi.toModify(siPixelDigis.cpu,  
+    UseQualityInfo = True,                         
+    SiPixelQualityLabel = 'forRawToDigi',
+)
+from CalibTracker.SiPixelESProducers.SiPixelQualityESProducer_cfi import siPixelQualityESProducer
+siPixelQualityRawToDigi.toModify(siPixelQualityESProducer, 
```

Comment:
modifiers have to be applied in the place of the module definitions.
Please remove here and apply in CalibTracker/SiPixelESProducers/python/SiPixelQualityESProducer_cfi.py